packages/cdk8s-cli/test/import/fixtures/crossplane.yaml

```diff
@@ -0,0 +1,969 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
What's the rationale for adding this?

Reply:
@eladb just wanted to make sure that some key crossplane CRDs passed all the cdk8s CRD import tests.

now that I've verified that I can remove if you like or leave it to ensure they continue to pass. it does bump the unit test time by a few seconds, so happy to remove.

Reply:
> Looks good. Feels like we should add some form of unit test?

@eladb that works ... what's your preference
a. full `import-k8s-test.ts` style test with multiple versions tested and validated against the snapshot (would increase unit test times)
b. simpler regex test similar to import-k8s-util.test.ts?

Reply:
Happy to add it, but let's do it in a separate PR?

Reply:
sounds good 👍